src/server/routes/api/document/reach.js

```diff
@@ -351,7 +354,7 @@ module.exports = {
               }
             }
 
-            if( intn.entries.length >= 2 ){
+            if( intn.entries.length >= 2 && intn.completed ){
```

Comment:
This line is what I mentioned at the end of PR comment. I can remove the extra check if preferred.

Reply:
One option would be removing this extra check from here but throwing error if the interaction is not complete.

Reply:
This is probably fine for now.  Maybe in future we could log these miss cases, but throwing an error might be too much

Reply:
Yes, I agree it would be breaking.